### ISSUE TITLE
Allow more flexible fileExtension

### ISSUE THREAD

**Issue by tracker1** [Gi60s/rolling-file#9]
I was wanting to have more info in the file extension (after the date part.

    fileExtension: `.${process.pid}.log`,

It would be nice, if the file extension offered that option.

```
OSError EVALID: Invalid configuration value for property: fileExtension. This value must be a string with only letters or numbers. Received: .28772.log
```

As it is, if it's part of the prefix, then logs for a given date would be separated by process id, instead of together.

**Comment by Gi60s** [comment]
This would probably be pretty easy to do. I'll take a look at it and get back to you. If you don't hear from me by the end of the week then post a new comment here to remind me. :smile:

**Comment by Gi60s** [comment]
Fixed in npm package version 1.0.11
